### DISCUSSION TITLE
Creating audit logs for fields which have a relation to the main audit model.

### DISCUSSION THREAD

**Discussion by HaglundOls** [owen-it/laravel-auditing#856] [Q&A] [answered]
I have a model, for example an user, on which i have implemented auditing and it works perfectly. But on the page where you can edit the values for the user there are also fields that have relations to the user. For example I have a model for comments about the user which are a separate model. The comments are also used for other models such as posts etc. How would i implement auditing for the comment also but only when they are changed or made on the edit user view? To clarify i don't want to create a log for all comments, only those who are for the user. 

Thanks!

**Comment by angeljqv** [comment, marked as answer]
https://github.com/owen-it/laravel-auditing/discussions/790#discussioncomment-6846951

Enabling or disabling auditing
